browser-extension/mv3/src/service-worker/services/clientHandler.ts

```diff
@@ -1,35 +1,44 @@
 import { generateUrlPattern } from "../../utils";
 import { WEB_URL, OTHER_WEB_URLS } from "../../../../config/dist/config.build.json";
-import { isExtensionEnabled } from "./utils";
+import { isExtensionEnabled } from "../../utils";
 import { Variable, onVariableChange } from "../variable";
 import { RequestRulePair, ResponseRulePair, RuleType } from "common/types";
 import rulesStorageService from "../../rulesStorageService";
 
-const CLIENT_SCRIPT_ID = "client-script";
 const excludeMatchesPatterns = [WEB_URL, ...OTHER_WEB_URLS].map(generateUrlPattern).filter((pattern) => !!pattern);
 
+const CLIENT_SCRIPTS: chrome.scripting.RegisteredContentScript[] = [
+  {
+    id: "client-script-ajaxInterceptor",
+    js: ["client-scripts/ajaxRequestInterceptor.js"],
+    world: "MAIN",
+    allFrames: true,
+    persistAcrossSessions: false,
+    matches: ["http://*/*", "https://*/*"],
+    runAt: "document_start",
+    excludeMatches: excludeMatchesPatterns,
+  },
+  {
+    id: "client-script-sessionRecorder",
+    js: ["client-scripts/sessionRecorderHelper.js"],
+    world: "MAIN",
+    persistAcrossSessions: false,
+    matches: ["http://*/*", "https://*/*"],
+    runAt: "document_start",
+  },
+];
+
```

Comment:
Can you move rest of the cleanup in a separate PR. PR should contain only what is specified in the title of the PR